ui/raidboss/raidboss.js

```diff
@@ -52,6 +54,18 @@ UserConfig.getUserConfigLocation('raidboss', function(e) {
       console.log('Enabling audio via query parameter');
   }
 
+  let PlayerNameOverride = params.get('player');
+  if (PlayerNameOverride !== null) {
+    Options.PlayerNameOverride = PlayerNameOverride;
+    console.log('Enabling player name override via query parameter, ' + PlayerNameOverride);
+    if (Options.audioAllowed) {
+      // include the tts engine script
+      // re-using this since it's already created
+      UserConfig.appendJSLink('browser_tts_engine.js');
```

Comment:
This gets parsed and run after gPopupText is created below, and so it's never included in time for the constructor.  Can you just include it always in the raidboss.html and emulator html?  I don't think load time is a serious concern here.

Reply:
This is fixed in my local git now. It was originally conditional because the script had side effects, but that version of the script never made it into the git tree.